.env

```diff
@@ -0,0 +1,5 @@
+REACT_APP_IMAGE_URL="https://apps.perseids.org/woodhouse/android-chrome-512x512.png"
+REACT_APP_SITE_NAME="Woodhouse English-Greek Dictionary"
+REACT_APP_TITLE="Woodhouse English-Greek Dictionary"
+REACT_APP_DESCRIPTION="This is the entire English-Greek Dictionary: A Vocabulary of the Attic Language by S. C. Woodhouse "

```

Comment:
Let's change this text to `English-Greek Dictionary: A Vocabulary of the Attic Language by S. C. Woodhouse`